core/templates/dev/head/pages/exploration_editor/EditorServices.js

```diff
@@ -85,28 +87,26 @@ oppia.factory('explorationData', [
           // changes applied. This makes a force-refresh necessary when changes
           // are discarded, otherwise the exploration-with-draft-changes
           // (which is cached here) will be reused.
-          return $http.get(explorationDataUrl, {
-            params: {
-              apply_draft: true
-            }
-          }).then(function(response) {
+          return EditableExplorationBackendApiService.fetchExploration(
```

Comment:
In EditableExplorationBackendApiService there is only Load to perform a fetch. A bit different from the load capability in ReadOnlyExplorationBackendApiService. Does this seem ok to you?